Code/GraphMol/FileParsers/MaeWriter.cpp

```diff
@@ -221,6 +223,61 @@ int bondTypeToOrder(const Bond& bond) {
   }
 }
 
+static bool is_double_any_bond(const RDKit::Bond& b)
+{
+    if (b.getBondType() == RDKit::Bond::DOUBLE) {
+        if (b.getStereo() == RDKit::Bond::BondStereo::STEREOANY ||
+            b.getBondDir() == RDKit::Bond::EITHERDOUBLE) {
+            return true;
+        }
+
+        // Check v3000/v2000 stereo either props
+        auto has_either_prop = [&b](const auto& prop, const int& either_value) {
+            return b.hasProp(prop) && b.getProp<int>(prop) == either_value;
+        };
+
+        return has_either_prop(RDKit::common_properties::_MolFileBondCfg, 2) ||
+               has_either_prop(RDKit::common_properties::_MolFileBondStereo, 3);
+    }
+    return false;
+}
+
+static void copyAtomNumChirality(const ROMol& mol, mae::Block& stBlock) {
+  // This property tells Schrodinger software that the stereo and chirality in
+  // the mae file are valid
+  stBlock.setIntProperty(MAE_STEREO_STATUS, 1);
+
+  // Set atom numbering chirality
+  int chiralAts = 0;
+  for (const auto at : mol.atoms()) {
+    std::string atomNumChirality;
+    if (at->getChiralTag() == Atom::CHI_TETRAHEDRAL_CW) {
+      atomNumChirality = "ANR";
+    } else if (at->getChiralTag() == Atom::CHI_TETRAHEDRAL_CCW) {
+      atomNumChirality = "ANS";
+    } else {
+      continue;
+    }
+    chiralAts++;
+    std::string propName = mae::CT_CHIRALITY_PROP_PREFIX + std::to_string(chiralAts);
```

Comment:
I guess `std::fmt` is C++20. Otherwise I'd suggest using it here and below.

Reply:
could use boost::format instead. That's header only, results in reasonably nice looking code, and is already used in the RDKit:
https://github.com/rdkit/rdkit/blob/81e9cb23868c7d8e6fe391b7809acfedae306661/Code/GraphMol/FileParsers/XYZFileWriter.cpp#L40